commands/account/remove.ts

```diff
@@ -50,7 +50,11 @@ export async function handler(
 
   trackCommandUsage('accounts-remove', undefined, account.accountId);
 
-  const currentDefaultAccount = getConfigDefaultAccount();
+  let currentDefaultAccount;
+
+  try {
+    currentDefaultAccount = getConfigDefaultAccount();
```

Comment:
Not a blocker, but I wonder if it would be worthwhile to support a version of popular utils like this one where they don't throw. Like adding an argument to it that lets it return null instead of throwing for situations like this one.

Reply:
Yeah, definitely more instances where we don't want this to throw than I thought. I think I'm going to stick with this for now, but if it ends up being an issue I'll add a non throwing version. Want to keep the LDL exports as uncluttered as possible

Reply:
I've run into this a few more times so going to add a `getConfigDefaultAccountIfExists`. Might switch over to that in a different PR though